src/aap_eda/api/serializers/activation.py

```diff
@@ -141,6 +152,7 @@ class ActivationCreateSerializer(serializers.ModelSerializer):
     sources = serializers.ListField(
         required=False,
         allow_null=True,
+        child=serializers.IntegerField(),
```

Comment:
I presume the `child` is an integer rather than a serialized representation 'cause `ActivationCreateSerializer`  is internal and we only need the `id`?